desktop/utils/domain/auctions/urls.js

```diff
@@ -0,0 +1,17 @@
+const sd = require('sharify').data
+
+/** Get the live auction url with /login appended when a user is present
+ * @param {String} id - auction id/slug
+ * @param {Object} options
+ * @param {Boolean} options.isLoggedIn - whether there is a user present
```

Comment:
Super small (but good to know) -- with [jsdoc syntax](http://usejsdoc.org/tags-param.html) you wrap optional parameters in brackets. So it would be 

`[options] ... ` 
`[options.isLoggedIn] ...`